grpc/rust/BUILD

```diff
@@ -16,13 +16,15 @@
 #
 
 load("@rules_rust//rust:defs.bzl", "rust_library")
-load("@rules_rust//cargo:cargo_build_script.bzl", "cargo_build_script")
+load("@vaticle_bazel_distribution//crates:rules.bzl", "assemble_crate", "deploy_crate")
+load("@vaticle_dependencies//distribution:deployment.bzl", "deployment")
 load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+load("@vaticle_dependencies//builder/grpc/rust:compile.bzl", "rust_tonic_compile")
 
-cargo_build_script(
+rust_tonic_compile(
     name = "typedb_protocol_src",
-    srcs = ["build.rs"],
-    data = [
+    packages = ["typedb.protocol"],
```

Comment:
The list of Protobuf packages in the `.proto` files. `rust_tonic_compile` needs this info to know what output files it is expected to generate.